app/controllers/poms_controller.rb

```diff
@@ -38,13 +38,17 @@ def update
 
   def my_caseload
     @pom = PrisonOffenderManagerService.get_signed_in_pom_details(current_user)
-    @allocations = PrisonOffenderManagerService.get_allocated_offenders(@pom.staff_id)
-    @new_cases = PrisonOffenderManagerService.get_new_cases(@pom.staff_id)
+    if @pom.present?
```

Comment:
The breadcrumbs in this file (for my_caseload and new_cases both have a hard-coded id of 1, but this can be removed as the ID isn't needed).